RELEASE_NOTES.md

```diff
@@ -28,211 +28,216 @@ You are strongly advised to take a backup first.
 
-- Add support for HEIF/HEIC file format.
+- The automatic mask tuning has been improved in the tone equalizer module.
 
-- Add support for ARM64/Apple M1 as build target.
+- Tiling has been enabled in the color balance rgb, diffuse and filmic rgb modules
+  to allow very large images to be processed.
 
-- The scopes module (previously named histogram) can be moved to the
-  left panel.
+- When scanning for updated XMP files, the synchronization window has been improved
+  to offer more choices as to how the database/XMP files should be updated.
 
-- Add preference to invert the behavior of mouse scroll up/down on
-  masks. At the same time, and for consistency, the scroll-up has been
-  set to increase the masks attributes.
+- Added support for HEIF/HEIC file format.
 
-- Add timestamp in camera import dialog for consistency with other
-  import dialog.
+- Added support for ARM64/Apple M1 as a build target.
 
-- Show current modules order in the module lib header to save one line
-  in GUI.
+- Added a preference to invert the behavior of mouse scroll up/down on drawn
+  mask attributes. At the same time, and for consistency, the scroll-up action has been
+  set to increase all mask attributes by default.
 
-- Display hue in degree in the split toning module for consistency
-  with other modules.
+- Added timestamp in camera import dialog for consistency with other
+  import dialog.
 
-- The rejected images in lighttable are dimmed for them to be better
-  distinguishable.
+- The current module order is now shown in the module order module header to 
+  save space in the GUI.
 
-- Remember the last selected Piwigo album in the export module.
+- The split toning module now displays the hue in degree for consistency
+  with other modules.
 
-- Add vertical waveform in scopes.
+- Rejected images in the lighttable view are now dimmed for clarity.
 
-- Use a new magic wand icons for automatic actions instead of the
-  color picker one. This is used for masks auto-tune in the Tone
-  Equalizer module.
+- The last selected Piwigo album is now remembered in the export module.
 
-- Add HSV reading in color picker.
+- New "magic wand" icons are now used for the auto-tune actions in
+  the tone equalizer module.
 
-- For advanced users and developers expose OpenCL building options in
+- For advanced users and developers, OpenCL build options are now exposed in
   darktablerc.
 
-- Move the module order entry in the copy/paste dialog at the end for
-  better usability as this option is rarely used.
-
-- The collect module offers some new presets based on image time to
-  complement the current ones based on the import time.
+- The "module order" entry is now moved to the end of the copy/paste dialog 
+  for better usability, as this option is rarely used.
 
-- Add places section into the import dialog and list there the default
-  folders (Home, Document, Images), the mounted drives and other
-  folders added by users for quick access.
```

Comment:
FYI I removed this as these features are all present in 3.6.1. I seem to recall some work around mounting and unmounting but I don't remember that it had much impact on how it worked in normal use.